src/constants.rs

```diff
@@ -31,4 +34,19 @@ pub const ENV_IMAGE_DIR: &str = "CRTOOLS_IMAGE_DIR";
 pub const IMG_STREAMER_CAPTURE_SOCKET_NAME: &str = "streamer-capture.sock";
 
 /// CONFIG_FILE is used to load checkpoint/restore parameters.
-pub const CONFIG_FILE: &str = "criu-coordinator.json";
\ No newline at end of file
+pub const CONFIG_FILE: &str = "criu-coordinator.json";
+
+/// Acknowledgment message sent to clients when an operation is successful.
+pub const MESSAGE_ACK: &str = "ACK";
+/// Synchronization message to indicate that a local checkpoint is ready.
+pub const MESSAGE_SYN: &str = "SYN";
+/// Acknowledgment message for successful receipt of an image chunk.
+pub const MESSAGE_IMG_ACK: &str = "IMG_ACK";
+/// Error message to signal a timed out during connection or readiness check.
+pub const MESSAGE_TIMEOUT: &str = "Timeout";
+/// Error message when a client dependency is not connected.
+pub const MESSAGE_NOT_CONNECTED: &str = "Not connected";
```

Comment:
Given that the two messages below this are lowercase, we would want to keep this consistent.

```suggestion
pub const MESSAGE_TIMEOUT: &str = "timeout";
/// Error message when a client dependency is not connected.
pub const MESSAGE_NOT_CONNECTED: &str = "not connected";
```